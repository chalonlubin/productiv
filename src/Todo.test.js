import React from "react";
import { render} from "@testing-library/react";
import Todo from "./Todo";

describe("Todo Tests", function () {
  const todo = {
    id: 1,
    title: "Test-Title",
    description: "Test-Description",
    priority: 1 };

  it("renders without crashing", function () {
    render(<Todo todo={todo} />);
  });

  it("contains expected title", function () {
    const result = render(<Todo todo={todo} />);
    expect(result.queryByText("Test-Title")).toBeInTheDocument();
  });

  it("matches snapshot", function () {
    const { container } = render(<Todo todo={todo}/>);
    expect(container).toMatchSnapshot();
  });
});