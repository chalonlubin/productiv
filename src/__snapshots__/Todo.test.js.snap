// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Tests matches snapshot 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        Test-Title
      </b>
       
      <small>
        (priority: 
        1
        )
      </small>
    </div>
    <div>
      <small>
        Test-Description
      </small>
    </div>
  </div>
</div>
`;
